feat: new metadata models endpoints

lukeocodes opened the pull request:
adds new metadata get model endpoints to the JS SDK

naomi-lgbt commented on test/manage_rest.test.ts:
```diff
@@ -228,4 +228,23 @@ describe("making manage REST requests", () => {
+
+    assert.isNull(error);
+    assert.isNotNull(result);
+    // Add your assertions for the response data here
```
Do we want to test that a specific property is present, like we do in the other tests?

lukeocodes replied:
shhhhh i hate this logic 

but yes we do :(

naomi-lgbt reviewed (approved):
This looks good! One non-blocking question.

jpvajda commented:
@naomi-lgbt  @lukeocodes  How close are we to getting this merged in?

lukeocodes commented:
@naomi-lgbt or @dvonthenen if you could review/approve. I have updated the tests to actually check the response body as requested, and commits now squashed

cc @jpvajda
